cppcache/integration/test/BasicIPv6Test.cpp

```diff
@@ -53,7 +55,11 @@ std::shared_ptr<Region> setupRegion(Cache& cache) {
  * Example test using 2 servers and waiting for async tasks to synchronize using
  * furtures.
  */
+#ifdef WITH_IPV6
+TEST(BasicIPv6Test, queryResultForRange) {
+#else
 TEST(BasicIPv6Test, DISABLED_queryResultForRange) {
```

Comment:
If I recall this test was disable because it relies on very specific address existing on the host or some specific address formatting. Something about it was not portable. We should just fix that.

Reply:
I think this comment got switched with the one about QueueConnectionRequestTest.  QueueConnectionRequestTest is indeed not portable, but I didn't quite figure out how to deal with the different address formatting